doc/modules/model_evaluation.rst

```diff
@@ -49,36 +49,36 @@ Common cases: predefined values
 For the most common use cases, you can designate a scorer object with the
 ``scoring`` parameter; the table below shows all possible values.
 All scorer objects follow the convention that **higher return values are better
-than lower return values**.  Thus the returns from mean_absolute_error
-and mean_squared_error, which measure the distance between the model
-and the data, are negated.
+than lower return values**.  Thus metrics which measure the distance between
+the model and the data, like mean_squared_error, are implemented as
```

Comment:
Here `mean_squared_error` refers to the metric and `neg_mean_squared_error` to the scorer. So I linked `mean_squared_error` to the metric, I think it makes it clearer, thansk for the tip!

I think you measure the quality of the model (via the predictions), so I vote to stick with what was already there.